src/components/Camera/index.js

```diff
@@ -64,8 +68,9 @@ const CameraPure = ({
         <Webcam
           className={style.video}
           audio={!!video}
-          height={cameraHeight}
+          height={idealCameraHeight || cameraHeight}
```

Comment:
I think this PR is a good chance to normalize constant naming convention, like using `CAPITALISED_SNAKE_CASE`'s instead of `camelCase`'s.
```suggestion
          height={idealCameraHeight || FALLBACK_CAMERA_HEIGHT_IN_PIXELS}
```